crypto/ess/build.info

```diff
@@ -1,3 +1,8 @@
 LIBS=../../libcrypto
-SOURCE[../../libcrypto]= \
-        ess_lib.c ess_asn1.c ess_err.c
+
+IF[{- !$disabled{'cms'} and !$disabled{'ts'} -}]
```

Comment:
This looks like a wrong build condition. Should there be `or` instead of `and`? But thinking about it even more - there are public functions implemented in ess_lib.c and they do not seem to be ifdefed out in the ess.h public header with NO_CMS and NO_TS so I do not think this is right at all.

Reply:
`ess_lib.c` methods have no caller when building with `no-cms` and `no-ts`.
What's why the `and` condition.
As these methods are internals, I see no need to ifdefed them in ess.h

Reply:
Ah, I did not realize these methods are internals-only.